CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.4-dev2
+
+* Add 
```

Comment:
You can add your bullet under `0.3.3-dev1` and then change the version to `0.3.3-dev2`. We'll move to `0.3.4-devx` once we do the `0.3.3` release.